src/Primary_Extension/org/lobobrowser/primary/ext/AddressField.java

```diff
@@ -75,11 +76,17 @@ public void actionPerformed(final ActionEvent event) {
       }
     });
 
-    getInputMap(JComponent.WHEN_IN_FOCUSED_WINDOW).put(KeyStroke.getKeyStroke("ctrl L"), "edit URL");
+    if (System.getProperty("os.name").toLowerCase().startsWith("mac os x")) {
+      getInputMap(JComponent.WHEN_IN_FOCUSED_WINDOW).put(KeyStroke.getKeyStroke(KeyEvent.VK_L, Toolkit.getDefaultToolkit().getMenuShortcutKeyMask()), "edit URL");
+    } else {
+      getInputMap(JComponent.WHEN_IN_FOCUSED_WINDOW).put(KeyStroke.getKeyStroke("ctrl L"), "edit URL");
+    }
+    final JComboBox<String> combo = this;
     getActionMap().put("edit URL", new AbstractAction() {
 
       public void actionPerformed(final ActionEvent e) {
         requestFocus();
+        combo.getEditor().selectAll();
```

Comment:
You don't need the combo variable here. Just `getEditor().selectAll()` will work.